test/test_unidler.py

```diff
@@ -0,0 +1,253 @@
+from io import BytesIO
+from unittest.mock import MagicMock, call, patch
+
+import pytest
+
+import unidler
+from unidler import (
+    IDLED,
+    IDLED_AT,
+    INGRESS_CLASS,
+    RequestHandler,
+    UNIDLER,
+)
+
+
+HOSTNAME = 'test.host.name'
+
+
+@pytest.yield_fixture
+def client():
+    client = MagicMock()
+    with patch('unidler.client', client):
+        yield client
+
+
+@pytest.fixture
+def deployment():
+    deployment = MagicMock()
+    deployment.spec.replicas = 0
+    deployment.metadata.labels = {
+        IDLED: 'true',
```

Comment:
I couldn't miss something here, don't we prefix these annotations with a namespace in the idler?

Reply:
Oh I see, these are constants with the usual prefixes, I was a bit confused about the lack of quotes in the keys actually :P - ignore me